gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,99 @@
+//tag::intro1[]
+
+Flex Gateway offers the following CLI command to test the health of a Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either a status code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
+
+//end::intro1[]
+
+//tag::intro2[]
+By using the liveness check command, you can either configure a Kubernetes liveness probe or a Docker health check command to restart a failed Flex Gateway.
+
+For Flex Gateways running on Linux, you can still run the liveness check command. However, there is not an easy to automate the Linux machine to restart when the command returns an unhealthy status.
+
+//tag::intro2[]
+
+//tag::k8s-liveness-probe[]
+== Configure a Kubernetes Liveness Probe
+
+For Flex Gateways running on Kubernetes, you can configure a Kubernetes liveness probe to monitor and restart a Flex Gateway pod when unhealthy. To learn more about Kubernetes probes, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/[Configure Liveness, Readiness and Startup Probes^].
+
+To enable a liveness probe, update you Helm chart with the following configuration:
+
+[source,helm]
+----
+livenessProbe:
+ exec:
+   command:
+   - flexctl
+   - probe
+   - --check=liveness
+ initialDelaySeconds: 10
+ periodSeconds: 10
+ failureThreshold: 5
+ timeoutSeconds: 1
+----
+
+[cols="1,2"]
+|===
+| Parameter | Description
+
+| `exec.command` | The command to execute to test liveness
+| `initialDelaySeconds` | The time in seconds to wait after startup before executing the first liveness test
+| `periodSeconds` | The period in seconds between each liveness check
+| `failureThreshold` | The number of failed liveness tests before the Kubernetes pod is restarted
+| `timeoutSeconds` | The time in seconds before the test times out
+|===
+
+To learn more about updating a Helm chart, see xref:gateway::flex-gateway-k8-change-helm-settings.adoc[Update a Helm Chart].
+
+//end::k8s-liveness-probe[]
+
+
+
+//tag::docker-liveness-probe[]
+== Docker Liveness Probe
+
+For Flex Gateways running on Docker, you can configure a health check using flags in the `docker run` command.
+
+After adding the relative Flex Gateway registration information from xref:gateway::flex-{page-mode}-reg-run.adoc[], execute the `docker run` command with the following flags configured:
+
+
+[source,ssh]
+----
+docker run --health-retries=5 \
+--health-interval=10s \
+--health-start-period=10s \
+--health-timeout=1s \
+--health-cmd="flexctl probe --check=liveness" \
+--entrypoint flexctl \
+-v "$(pwd)":/registration -u $UID mulesoft/flex-gateway \
+register \
+#REPLACE WITH RELATIVE FLEX GATEWAY REGISTRATION
+my-gateway
```

Comment:
Actually, the command that we should use here is the one to start flex, not to register https://docs.mulesoft.com/gateway/latest/flex-local-reg-run-up#start-command